programs/clearing_house/src/controller/orders.rs

```diff
@@ -1204,9 +1224,11 @@ pub fn fulfill_order_with_match(
     maker_stats: &mut UserStats,
     maker_order_index: usize,
     maker_key: &Pubkey,
-    filler: Option<&mut User>,
-    filler_stats: Option<&mut UserStats>,
+    filler: &mut Option<&mut User>,
+    filler_stats: &mut Option<&mut UserStats>,
     filler_key: &Pubkey,
+    mark_price_before: u128,
```

Comment:
@0xbigz should we move updating mark price outside of the amm swap? think we talking about this before

Reply:
oh right, if two users match, it wont update twap?